skietbaan_fe/src/components/Groups.js

```diff
@@ -2,15 +2,14 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import './groups.css';
 import { withRouter } from 'react-router-dom';
-import { createGroups } from '../actions/postActions';
+import { createGroups,FetchGroups,groupDic } from '../actions/postActions';
```

Comment:
what does groupDic mean?

Reply:
groups dictionary

Reply:
Change the name accordingly please. Should not be vague.